src/main/java/org/black_ixx/playerpoints/locale/EnglishLocale.java

```diff
@@ -58,6 +58,7 @@ public String getTranslatorName() {
             this.put("command-take-description", "&8 - &d/points take &7- Take points from a player");
             this.put("command-take-usage", "&cUsage: &e/points take <player> <amount>");
             this.put("command-take-success", "&aTook &b%amount% &a%currency% from &b%player%&a.");
+            this.put("command-took-taken", "&b%amount% &e%currency% has been taken.");
```

Comment:
This should say `take` instead of `took`.

Reply:
oh yes soory